aptos-move/framework/aptos-stdlib/sources/signature.move

```diff
@@ -1,12 +1,37 @@
 /// Contains functions for [ed25519](https://en.wikipedia.org/wiki/EdDSA) digital signatures and for
 /// [Boneh-Lynn-Shacham (BLS) signatures](https://en.wikipedia.org/wiki/BLS_digital_signature)
 module aptos_std::signature {
-    /// Return `true` if the bytes in `public_key` is a valid bls12381 public key and
-    /// passed the PoP check.
+    use std::option::Option;
+
+    /// Given a vector of serialized public keys, combines them into an aggregated public key which can be used to verify
+    /// multisignatures using `bls12381_verify_signature`.
+    /// Returns 'None' if no public keys are given as input. This function assumes that the caller validated all input
+    /// public keys using `bls12381_validate_pubkey`.
+    /// Does not abort.
+    native public fun bls12381_aggregate_unvalidated_pubkeys(
```

Comment:
the comment says assumes caller validated all input but the function name says unvalidated pubkeys?

Reply:
Oopsie, will fix.